test/Microsoft.AspNet.Mvc.Core.Test/Formatters/OutputFormatterTests.cs

```diff
@@ -50,8 +50,8 @@ public static IEnumerable<object[]> SelectResponseCharacterEncodingData
             // Arrange
             var mockHttpContext = new Mock<HttpContext>();
             var httpRequest = new DefaultHttpContext().Request;
-            httpRequest.Headers["Accept-Charset"] = acceptCharsetHeaders;
-            httpRequest.ContentType = "application/acceptCharset;charset=" + requestEncoding;
+            httpRequest.Headers[HeaderNames.AcceptCharset] = acceptCharsetHeaders;
+            httpRequest.Headers[HeaderNames.Accept] = "application/acceptCharset;charset=" + requestEncoding;
```

Comment:
This isn't the same test anymore. Now giving the formatter something it should pay attention to. Suggest just removing the `ContentType` assignment; don't add an `Accept` header.